features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,15 @@
-# put step definitions in this file
\ No newline at end of file
+When(/^I visit the site$/) do
+  visit '/'
+end
+
+Then(/^I should see "([^"]*)" is a student at Craft Academy$/) do |arg1|
+  expect(page).to have_content
+end
```

Comment:
This step doesn't test for anything at all. In examples you can see from slow food,  the line looks like `expect(page).to have_content content` -> `content` is a variable that comes from the first line. Here you are using `arg1`. At the very least, you would want the line to be `expect(page).to have_content arg1`, but you should rename that variable to what you are actually looking for - `content` or `text`.